npm/bin/ssc.js

```diff
@@ -23,6 +23,11 @@ async function main () {
     ...process.env
   }
 
+  if (!await installation.firstTimeExperienceSetup()) {
+    // FTE not completed satisfactorally, or 'setup' was ran externally
```

Comment:
should we log something here?

Reply:
@chicoxyzzy The user will be able to see the results of `install.sh` or `install.ps1` at this time.